### PR TITLE
add new "OBO Dashboard" section to FAQ menu

### LINKED ISSUE #1930: Improve FAQ menu
In #1280, these points were made:

> *  There is this page https://obofoundry.org/faq (not linked from anywhere)
> * Which points to a page https://obofoundry.org/join.shtml (that does not exist)

I think https://obofoundry.org/faq is obsolete. There is a FAQ menu on obofoundry.org that includes some of the info that's on that orphan faq page. I went through it to see if there was anything else on it that should be added to the FAQ menu (see below).

(By the way, https://obofoundry.org/faq not only isn't linked anywhere, it doesn't even come up in a google search. https://www.google.com/search?q=%22How+do+I+join+the+OBO+Foundry%3F%22&oq=%22How+do+I+join+the+OBO+Foundry%3F%22&aqs=chrome..69i57.925j0j7&sourceid=chrome&ie=UTF-8)

I went through https://obofoundry.org/faq. I think there is some useful info to mine from it and put elsewhere; then that obsolete page can be deleted. Below I comment on each section of /faq and suggest how I think we should proceed.

> How do I join the OBO Foundry?
Joining is easy! Just follow the steps on our join page [link 404s]. We look forward to hearing from you.

Not in main FAQ. Not clear what 'join' even means. "Contribute" section of FAQ has "How do I submit my ontology", and there's a Participate menu with mailing lists, Slack, etc., so I don't think there's any content here we need to merge in. **Action: none** (no content needs to be transferred from orphan page).

> How should I cite the OBO Foundry?
Smith B....(2007). "The OBO Foundry: coordinated evolution of ontologies to support biomedical data integration", Nature Biotechnology 25, 1251 - 1255.

That is the old publication. There's no "how to cite OBO Foundry" in main FAQ -- Citation section just has "How do I cite an ontology?". There is a Citation menu that includes "Cite the OBO Foundry" (which points to the new paper). **Action: add "How do I cite the OBO Foundry?" to main FAQ to make the info easier to find.** (see below for summary of action items)

> Are the ontologies in the OBO Foundry available only in OBO format?
No! Here is a definition of the OBO language in terms of OWL semantics [links to https://www.bioontology.org//wiki/Index.php/OboInOwl:Main_Page, which is empty]. You can download any OBO ontology in various formats--including OWL--here [raw link to http://ontologies.berkeleybop.org/].

There is a page https://obofoundry.org/faq/where-is-the-obo-file.html -- maybe this info could be added to that page? There's also http://robot.obolibrary.org/convert about converting between e.g. OWL and OBO. **Action: I could use some guidance about what to do here.**

> Is there a tool that will let me create or edit OBO ontologies?
OBO-Edit, an open source Java tool, offers a convenient visual interface for creating, editing, and browsing ontologies. It was initially created to work on OBO-format ontologies, but is currently being extended to handle other ontology formats, such as OWL.

This seems very out of date. **Action: don't preserve this text. Instead, add a Q to "Guides" section of FAQ: "What tools are there for creating, editing and working with OBO ontologies?" -> https://obofoundry.org/resources**

> How should we resolve conflicts when ontologies overlap?
Make every attempt to resolve inconsistencies and merge the ontologies. Tools are available for aiding this process (such as OBOL and Prompt) and the community can be used to help clarify best practices. Also, face-to-face meetings can helpful to resolve details that cannot be covered through email.

Not much content here. It seems like maybe this topic should be covered in one of the principles, though I'm not sure which. **Action: Not sure what to do here.**

-------------------
In addition to looking at the orphan FAQ, I went through the current FAQ menu on obofoundry.org. Below are my suggested steps (with a few questions).

#### First, for reference, here is the current FAQ menu:
Policy
- What are the OBO Foundry Principles?
- What is the OBO ID Policy?
- My ontology is not ready for OBO Foundry, can I reserve a prefix anyway?
Citation
- How do I cite an ontology?
Terms
- How do I request a new ontology term?
Contribute
- How do I submit my ontology?
- How do I request a review of my ontology?
- Where can I find ontology reviews?
Ontology display
- How do I edit the metadata for my ontology?
- How do I add a custom browser for my ontology?
- What is the build object in the ontology metadata?
- Why is there no OBO-Format file listed for my ontology?
- What does it mean for an ontology to be flagged as 'inactive', 'orphaned' or 'obsolete'?
Guides
- How should I manage my ontology using version control?
- How do I convert between formats (e.g. OBO to OWL format)?

My proposed changes to FAQ menu (in https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_includes/navbar.html):

- [x] Cite section: add Q "How do I cite the OBO Foundry?", A: https://obofoundry.org/registry/publications.html
(I made a PR that makes it clearer on that page how to cite the OBO Foundry: #1924)

> Contribute -> How do I request a review of my ontology? (https://obofoundry.org/docs/ReviewProcessGuidelines.html)
> Contribute -> Where can I find ontology reviews? (https://obofoundry.org/docs/CompletedReviews.html)

- [x]  I suggest we remove both of these from the FAQ. The second page (showing a few outdated ontology reviews) is kind of embarrassing, and as an all-volunteer organization, we don't have the resources to review already-admitted ontologies (as we've discussed on calls). Furthermore, the ontology dashboard automates a lot of the things that used to be done by human reviewers.

- [x] If we do remove those from the FAQ, we should consider also updating the issue submission template to remove the ontology review request.

Since the only thing left in the Contribute section would be "How do I submit my ontology?" I suggest we
- [x] move "How do I submit my ontology?" to Participate section and delete the Contribute section

- [x] Add Q to "Guides" section: "What tools are there for creating, editing and working with OBO ontologies?" -> https://obofoundry.org/resources

- [x] delete: Participate ->  Can I help improve the OBO portal? (points to https://obofoundry.org/faq/how-do-i-modify-website.html)
Not all pages even have an edit button, and it's a pretty fringe way to start editing the website anyway. My new Participate page (see PR #1928) points to this page.

- [x] Add to Participate section:
How can I get involved in the OBO Foundry? -> /docs/participate.html

- [ ] Add Dashboard somewhere - maybe a new section? "What is the OBO Dashboard?" -> http://dashboard.obofoundry.org/
"How can I assess my ontology using the OBO Dashboard?" -> **??? is there in fact a way to do that for an ontology that is not (yet) in OBO Foundry? I feel like there must be but I'm not sure what it is.**

- [x] The link for "How should I manage my ontology using version control?" points to https://douroucouli.wordpress.com/2014/01/08/creating-an-ontology-project/
**Is that 2014 blog post really the best and most up-to-date reference?**

Proposed change to Citation top-level menu:

- [x] Cite the OBO Foundry -> currently points straight to the publication itself, which isn't exactly a citation. I suggest pointing this to https://obofoundry.org/registry/publications.html (which has the citation as well as linking to the publication)

### PR DESCRIPTION
**Pull request by nlharris**
Along with #2014, fixes #1930

(sorry this is on a separate branch; i tried briefly to remember how to add (from the web interface) another PR and decided it wasn't worth spending more time on that)

### CODE REVIEW

**Comment by cthoyt** [comment]
This dropdown is getting pretty out of hand - it doesn't fit on a normal page anymore

**Comment by matentzn** [comment]
@cthoyt I agree. I think we should start considering making a faq.md page and having the menu item simply go to that page.

We will accept this PR here, but no more after that: https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2016

File: _includes/navbar.html [hunk 1]
```diff
@@ -118,6 +118,10 @@
             <li><a href="/faq/what-is-the-build-field.html">What is the <code>build</code> object in the ontology metadata?</a></li>
             <li><a href="/docs/OntologyStatus.html">What does it mean for an ontology to be flagged as 'inactive', 'orphaned' or 'obsolete'?</a></li>
             <li role="separator" class="divider"></li>
+            <li class="dropdown-header"><i>OBO Dashboard</i></li>
+            <li><a href="/faq/dashboard.html">What is the OBO Dashboard?</a></li>
+            <li><a href="/faq/dashboard.html">How can I assess my ontology using the OBO Dashboard?</a></li>
```
Comment (matentzn) [inline _includes/navbar.html:123]: Same link, yea?

Comment (nlharris) [reply _includes/navbar.html:123]: yes. same link. both answers are on that same page. i thought about adding an anchor but the page isn't very long.

**Review by matentzn — APPROVED** [review]

**Comment by nlharris** [comment]
the new menu items point to the new dashboard page, so they get a 404 until PR #2014 is merged.

**Pull request merged**